host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@10.0.1-cuda.cmake

```diff
@@ -80,8 +80,11 @@ set(MFEM_DIR "${TPL_ROOT}/mfem-4.2.0" CACHE PATH "")
 # Root directory for generated developer tools
 set(DEVTOOLS_ROOT "/usr/WS2/smithdev/devtools/blueos_3_ppc64le_ib_p9/2020_09_02_16_03_10/gcc-8.3.1" CACHE PATH "")
 
-# Docs dont work on blueos machines, there is a prompt that waits for user input
-set(ENABLE_DOCS OFF CACHE BOOL "")
+set(ENABLE_DOCS ON CACHE BOOL "")
+
+set(DOXYGEN_EXECUTABLE "${DEVTOOLS_ROOT}/doxygen-1.8.17/bin/doxygen" CACHE PATH "")
+
+set(SPHINX_EXECUTABLE "/collab/usr/gapps/python/build/spack-coralea.4/opt/spack/linux-rhel7-power8le/gcc-4.9.3/python-3.8.2-vgiumi4ushemn2ywaxcibgo3kw6yvtfj/bin/sphinx-build" CACHE PATH "")
```

Comment:
Do the docs work on blueos now?

Reply:
no... but i blocked it in the build scripts a while back because it broke everywhere and i commented it out in anger.